src/services/platform.service.js

```diff
@@ -51,3 +51,8 @@ export const getBaseProjectEnvironment = (projectPath: string) => ({
     path.delimiter +
     path.join(projectPath, 'node_modules', '.bin'),
 });
+
+export const getPlatformSpecificFolderName = () =>
```

Comment:
I think we should create a constant for this and the naming should be different so it's clear that it's just the caption of the open folder label. 
Something like `platformSpecificFolderCaption = isMac ? 'in Finder' : 'folder';`. 
I also think that we should move `in` to the caption as `Open folder` is better than `Open in folder`.